### PR TITLE
Lowered the reading level, and simplified some things

### PR DESCRIPTION
**Pull request by death7654**
this just makes it easier for newcomers, and non-english speakers to understand

please tell me if i got something wrong, of if i deleted too much

### CODE REVIEW

**Review by mine-man3000 — COMMENTED** [review]
> A Chrultrabook is a modified Chromebook that is able to run Windows, Linux, or even MacOS by using MrChromebox's Coreboot firmware. The purpose of this site is to provide documentation that is detailed, user-friendly, and has up to date information about the hardware, software, and firmware on Chromebooks
the "m" in "macOS"  is lowercase, Apple themselves use a lowercase m (slight nitpick though)
> Because of this, you **are** be expected to read, learn, and use Google. This is not a one click procedure.
slight grammatical error, "you **are** be expected" should either stay as "you will be expected" or changed to "you are expected"

File: docs/getting-started.md [hunk 1]
```diff
@@ -36,10 +36,12 @@ Please refer to [this chart](supported-devices.html) for a full list of supporte
   * Your iGPU
   * Your storage devices (HDD/SSD, eMMC, NVMe, etc.)
   * **Your boardname**
+       * For Example, The Boardname for Asus Chromebook C302CA is Cave
```
Comment (meghan06) [inline docs/getting-started.md:39]: Why is almost every word capitalized?

It should just be:
For example, the board name for the Asus Chromebook C302CA is CAVE

File: docs/getting-started.md [hunk 2]
```diff
@@ -36,10 +36,12 @@ Please refer to [this chart](supported-devices.html) for a full list of supporte
...
   * If you don't know where to find this info, take a look at the [System Info Page.](system-info.html)
 *  **A basic knowledge of command lines and how to use a terminal/command prompt.**
-  * We can't help you if you don't know how to cd to a directory or delete a file.
+  * We can't help you if you don't know how to go to a directory or delete a file from terminal.
```
Comment (meghan06) [inline docs/getting-started.md:42]: Should be "from a terminal"

File: docs/getting-started.md [hunk 3]
```diff
@@ -10,9 +10,9 @@ nav_order: 4
 
 | **Term** | **Description** |  
 | - | - |
-| chrultrabook | A modified Chromebook/box designed to run Windows, Linux, or even macOS |   
-| coreboot     | UEFI firmware firmware replacement |  
-| UEFI         | The Unified Extensible Firmware Interface (UEFI) is a specification that defines a software interface between an operating system and platform firmware. UEFI replaces the legacy Basic Input/Output System (BIOS) firmware interface originally present in all IBM PC-compatible personal computers, with most UEFI firmware implementations providing support for legacy BIOS services. UEFI can support remote diagnostics and repair of computers, even with no operating system installed. (source: Wikipedia) |  
+| Chrultrabook | A modified Chromebook or Chromebox designed to run Windows, Linux, or even macOS |   
+| Coreboot     | UEFI firmware firmware replacement |
```
Comment (meghan06) [inline docs/getting-started.md:14]: coreboot is all lowercase

File: docs/getting-started.md [hunk 4]
```diff
@@ -10,9 +10,9 @@ nav_order: 4
 
 | **Term** | **Description** |  
 | - | - |
-| chrultrabook | A modified Chromebook/box designed to run Windows, Linux, or even macOS |   
-| coreboot     | UEFI firmware firmware replacement |  
-| UEFI         | The Unified Extensible Firmware Interface (UEFI) is a specification that defines a software interface between an operating system and platform firmware. UEFI replaces the legacy Basic Input/Output System (BIOS) firmware interface originally present in all IBM PC-compatible personal computers, with most UEFI firmware implementations providing support for legacy BIOS services. UEFI can support remote diagnostics and repair of computers, even with no operating system installed. (source: Wikipedia) |  
+| Chrultrabook | A modified Chromebook or Chromebox designed to run Windows, Linux, or even macOS |
```
Comment (meghan06) [inline docs/getting-started.md:13]: Chrultrabook should be all lowercase too

File: docs/supported-devices.md [hunk 5]
```diff
@@ -6,12 +6,14 @@ nav_order: 7
 
 
 ## Supported Devices and Platforms
-Supported devices are listed in the table below, grouped by platform/family, roughly in chronological order. To reiterate, support is determined entirely and exclusively by the Board Name. Device Names are just for reference, and may be incomplete as many models/sub-models may use the same Board Name. Figure out the Board Name and then locate it in the table. If it's not listed, it's not supported.
+Supported devices are listed in the table below, grouped by platform/family. If your device is not listed, it is not supported. Support is determined entirely and exclusively by the Board Name. Use your Chromebook's Boardname to see if it has support.
```
Comment (meghan06) [inline docs/supported-devices.md:9]: >by the Board Name. Use your Chromebook's Boardname to see if it has support.

Board Name and Boardname? 

Also board name should be lower case

File: docs/unbricking.md [hunk 6]
```diff
@@ -60,7 +60,7 @@ Googling should locate a disassembly guide for most models. If you can't find on
 
 **Prepping to Flash**
 
-Once you have your device disassembled and flash chip located, time to boot up the flashing environment. Most any Linux setup should do as long as either flashrom is available from the distro's software repositories, or it's 64-bit x86 (in which case you can download a statically compiled build of flashrom from mrchromebox.tech). This guide will use a Ubuntu 23.04 live session booted from USB.
+Once you have your device disassembled and flash chip located, boot up the flashing environment. Most any Linux setup should do as long as either flashrom is available from the distro's software repositories, or it's 64-bit x86 (in which case you can download a statically compiled build of flashrom from mrchromebox.tech). This guide will use a Ubuntu 23.04 live session booted from USB.
```
Comment (meghan06) [inline docs/unbricking.md:63]: I had issues flashing on Ubuntu 23.04, but Fedora was fine. I think flashrom was complaining of outdated dependencies, not sure though.

File: index.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ nav_order: 1
 
 ### What is a Chrultrabook and who is this guide for?
 
-A chrultrabook is a modified Chromebook designed to run Windows, Linux, or even macOS by utilizing MrChromebox coreboot firmware. The purpose of this site is to provide comprehensive and user-friendly documentation on hardware, firmware, and operating systems.
+A Chrultrabook is a modified Chromebook that is able to run Windows, Linux, or even MacOS by using MrChromebox's Coreboot firmware. The purpose of this site is to provide documentation that is detailed, user-friendly, and has up to date information about the hardware, software, and firmware on Chromebooks
```
Comment (meghan06) [inline index.md:11]: Chrultrabook and Coreboot should be lowercase.

MacOS should be macOS.

**Review by meghan06 — CHANGES_REQUESTED** [review]
See comments I made

File: docs/unbricking.md (hunk 6 above)
Comment (death7654) [reply docs/unbricking.md:63]: I didnt add that

**Review by death7654 — COMMENTED** [review]
Everything should be corrected

File: docs/getting-started.md [hunk 8]
```diff
@@ -10,9 +10,9 @@ nav_order: 4
 
 | **Term** | **Description** |  
 | - | - |
-| chrultrabook | A modified Chromebook/box designed to run Windows, Linux, or even macOS |   
+| chrultrabook | A modified Chromebook or Chromebox designed to run Windows, Linux, or even macOS |
```
Comment (WeirdTreeThing) [inline docs/getting-started.md:13]: ChromeOS device is more generic

File: docs/getting-started.md [hunk 9]
```diff
@@ -36,10 +36,12 @@ Please refer to [this chart](supported-devices.html) for a full list of supporte
   * Your iGPU
   * Your storage devices (HDD/SSD, eMMC, NVMe, etc.)
   * **Your boardname**
+       * For example, the boardname for Asus Chromebook C302CA is Cave
   * If you don't know where to find this info, take a look at the [System Info Page.](system-info.html)
 *  **A basic knowledge of command lines and how to use a terminal/command prompt.**
-  * We can't help you if you don't know how to cd to a directory or delete a file.
+  * We cannot help you if you do not know how to go to a directory or delete a file from a terminal.
```
Comment (WeirdTreeThing) [inline docs/getting-started.md:42]: Kinda contradicts itself, you need to know how commands like `cd` and `rm` works so simplifying makes no sense

File: docs/getting-started.md [hunk 10]
```diff
@@ -36,10 +36,12 @@ Please refer to [this chart](supported-devices.html) for a full list of supporte
...
 * A minimum of an 8GB USB drive or SD Card for creating a bootable OS installer.
+* A Chromebook running Ventoy (read [Ventoy](ventoy.html), or another computer running Balena Etcher, or Rufus (read [Making a Bootable USB](bootableusb.html)
```
Comment (WeirdTreeThing) [inline docs/getting-started.md:44]: A chromebook is already a prerequisite

File: docs/unbricking.md (hunk 6 above)
Comment (WeirdTreeThing) [reply docs/unbricking.md:63]: You are supposed to use the flashrom included in ubuntu's repos, so dependencies should not be an issue

File: index.md [hunk 11]
```diff
@@ -21,13 +21,13 @@ A chrultrabook is a modified Chromebook designed to run Windows, Linux, or even
...
+*   Installing an altOS on your Chromebook.
 *   Learning in-depth information about your Chromebook.
 
-Because of this, you will be expected to read, learn, and even use Google. This is not a one click procedure.
+Because of this, you **are** expected to read, learn, and use Google. This is not a one click procedure.
 
 --------------------------------------
 
-Finally, those who are still facing issues can ask for help in the [chrultrabook Discord](https://discord.com/invite/tkPTk5w).
+Finally, those who are still facing issues can ask for help in the [Chrultrabook Discord](https://discord.com/invite/tkPTk5w).
```
Comment (WeirdTreeThing) [inline index.md:31]: capital?

**Review by WeirdTreeThing — CHANGES_REQUESTED** [review]

File: docs/altos.md [hunk 12]
```diff
@@ -18,9 +18,9 @@ Pros:
 
 Cons: 
 * Audio driver is paid (10 USD) for Skylake (6th generation) and newer devices
-* Thunderbolt driver is paid (20 USD) for certain generations
+* Thunderbolt driver is paid (20 USD) for certain generation (**Only** for devices that have a Thunderbolt port)
```
[the remark this answers is not in the document]
Comment (death7654) [reply docs/altos.md]: done

File: docs/getting-started.md (hunk 10 above)
Comment (death7654) [reply docs/getting-started.md:44]: its good to be explicit, since some people flash without having a bootable flash drive

File: docs/altos.md [hunk 13]
```diff
@@ -18,9 +18,9 @@ Pros:
 
 Cons: 
 * Audio driver is paid (10 USD) for Skylake (6th generation) and newer devices
-* Thunderbolt driver is paid (20 USD) for certain generations
+* Thunderbolt driver is paid (20 USD) for devices that **have** them
```
Comment (meghan06) [inline docs/altos.md:21]: I don't like how this is worded, maybe something like:

Certain devices with Thunderbolt ports will need to purchase an additional driver ($20 USD)

File: docs/getting-started.md [hunk 14]
```diff
@@ -10,9 +10,9 @@ nav_order: 4
 
 | **Term** | **Description** |  
 | - | - |
-| chrultrabook | A modified Chromebook/box designed to run Windows, Linux, or even macOS |   
+| chrultrabook | A modified ChromeOS device designed to run Windows, Linux, or even macOS |   
 | coreboot     | UEFI firmware firmware replacement |  
-| UEFI         | The Unified Extensible Firmware Interface (UEFI) is a specification that defines a software interface between an operating system and platform firmware. UEFI replaces the legacy Basic Input/Output System (BIOS) firmware interface originally present in all IBM PC-compatible personal computers, with most UEFI firmware implementations providing support for legacy BIOS services. UEFI can support remote diagnostics and repair of computers, even with no operating system installed. (source: Wikipedia) |  
+| UEFI         | The Unified Extensible Firmware Interface (UEFI) is a new method by which OSes and platform firmware communicate, providing a BIOS alternative that uses only the information needed to launch the OS boot process. |
```
Comment (meghan06) [inline docs/getting-started.md:15]: I feel like the UEFI description can still be simplified.

File: docs/supported-devices.md [hunk 15]
```diff
@@ -6,12 +6,14 @@ nav_order: 7
...
 
-The chart below provides a full, authoritative list of all ChromeOS devices currently supported by MrChromebox's Firmware Utility Script, the firmware types available for each, and the hardware write-protect method used. In most cases, all of the devices in a given platform/family are supported identically, but sometimes there are outliers (esp with older or newer devices). The goal is to offer both RW_LEGACY and UEFI Full ROM firmware for all devices wherever possible, so you have the option of dual booting ChromeOS, or liberating your device completely :)
+The chart below provides a full list of all ChromeOS devices currently supported by MrChromebox's Firmware Utility Script, the firmware types available for each, and the hardware write-protect method used. 
+
+In most cases, all of the devices in a given platform/family are supported, but sometimes there are exceptions (especially with older or newer devices). The goal is to offer both RW_LEGACY and UEFI Full ROM firmware for all devices, so you may have the option of dual booting ChromeOS, or liberating your device completely :)
```
Comment (meghan06) [inline docs/supported-devices.md:16]: You should mention that EOL devices do not have RWL

**Review by meghan06 — CHANGES_REQUESTED** [review]

File: docs/altos.md (hunk 13 above)
Comment (death7654) [reply docs/altos.md:21]: yeah that sounds better, ill add that in

File: docs/getting-started.md (hunk 14 above)
Comment (death7654) [reply docs/getting-started.md:15]: im not sure how to, but i can delete like one word

**Review by death7654 — COMMENTED** [review]
all changes requested should be applied

**Comment by death7654** [comment]
i added things

**Review by ninelore — CHANGES_REQUESTED** [review]
Balena Etcher is suspected of violating GDPRT and should neither be used nor recommended

**Review by meghan06 — DISMISSED** [review]
Looks good to me

File: docs/altos.md [hunk 16]
```diff
@@ -18,9 +18,9 @@ Pros:
 
 Cons: 
 * Audio driver is paid (10 USD) for Skylake (6th generation) and newer devices
-* Thunderbolt driver is paid (20 USD) for certain generations
+* **Certain** devices with Thunderbolt ports will need to purchase an additional driver ($20 USD)
```
Comment (ethanaobrien) [inline docs/altos.md:21]: All devices with thunderbolt ports are paid.

Maybe word this in the same way as the `Audio driver is paid (10 USD) for Skylake (6th generation) and newer devices` line. The thunderbolt driver is needed on tigerlake (11th gen) and up if I'm not mistaken

File: docs/getting-started.md [hunk 17]
```diff
@@ -49,7 +51,9 @@ If you are facing a issue, please read the [FAQ](faq.html) first.
 {: .important } 
 > When asking for help, ask in English. We can't help you if we don't understand the question. Using a translator is fine.
 
-**Do not use manufacturer's model name or serial number when asking for help** (i.e: HP Chromebook 14a), it doesn't help with identifying the machine. Provide the boardname, otherwise your support request will be ignored. If you don't know where to find your boardname, take a look [here.](system-info.html)
+**Do not use manufacturer's model name or serial number when asking for help** (i.e: HP Chromebook 14a), it doesn't help with identifying the machine. Provide the boardname, otherwise your support request will be ignored. 
+
+If you don't know where to find your boardname, take a look [here.](system-info.html)
```
Comment (ethanaobrien) [inline docs/getting-started.md:56]: (doesnt really matter)

Should the period be part of the link?

File: index.md [hunk 18]
```diff
@@ -21,10 +21,10 @@ A chrultrabook is a modified Chromebook designed to run Windows, Linux, or even
 
 **The docs focuses on two main things:**
 
-*   Installing altOS on your Chromebook.
+*   Installing an altOS on your Chromebook.
 *   Learning in-depth information about your Chromebook.
 
-Because of this, you will be expected to read, learn, and even use Google. This is not a one click procedure.
+Because of this, you **are** expected to read, learn, and use Google. This is not a one click procedure.
```
Comment (ethanaobrien) [inline index.md:27]: This is my opinion (check with others before making this a reality) but I think we should either just bold this entire line or also bold "This is not a one click procedure"

**Review by ethanaobrien — CHANGES_REQUESTED** [review]

File: docs/altos.md (hunk 16 above)
Comment (WeirdTreeThing) [reply docs/altos.md:21]: Keep in mind not all of them have tbt, some are usb4 which also needs that driver.

File: docs/getting-started.md (hunk 17 above)
Comment (WeirdTreeThing) [reply docs/getting-started.md:56]: We should never include the . in the link

File: index.md (hunk 18 above)
Comment (WeirdTreeThing) [reply index.md:27]: bold

File: docs/altos.md (hunk 16 above)
Comment (meghan06) [reply docs/altos.md:21]: @ethanaobrien @death7654 

Maybe reword to `USB4/Thunderbolt driver is paid (20 USD) for TigerLake (11xxx) and newer`

File: docs/bootableusb.md [hunk 19]
```diff
@@ -31,7 +31,37 @@ For this guide, we will be using Rufus to create our bootable USBs.
...
+2. Plug in your USB
+3. Launch Balena Etcher from applications
+4. Click the big blue "Select" button
+5. A File Explorer window should pop up, find the ISO of your choice, then click `Open`
+6. Select your Flashdrive or Change your Flashdrive
+7. Click On Flash
+8. It may ask for adminstrator privileges, click on `allow`
+9. Profit
+
+### Flashing using Balena Etcher (Linux)
```
Comment (meghan06) [inline docs/bootableusb.md:53]: Can we create a shell script that does everything for you? Idk if that's a thing or not.

File: docs/getting-started.md [hunk 20]
```diff
@@ -36,10 +36,12 @@ Please refer to [this chart](supported-devices.html) for a full list of supporte
   * Your iGPU
   * Your storage devices (HDD/SSD, eMMC, NVMe, etc.)
   * **Your boardname**
+       * For example, the boardname for Asus Chromebook C302CA is Cave
```
Comment (meghan06) [inline docs/getting-started.md:39]: Cave -> ``Cave``

File: docs/installing-linux.md [hunk 21]
```diff
@@ -33,6 +33,8 @@ Ubuntu and Ubuntu-based distributions, such as Mint or ElementaryOS are unsuppor
 2. Plug the USB Drive into the Chromebook.
 3. Turn on the Chromebook, press `ESC` at the POST screen (when the boot logo appears), and select your USB to boot from. 
 4. Install as you would on any other computer.
+5. (Optional) Install Chultrabook Tools for fan controls, backlight controls, System Information and more
```
Comment (meghan06) [inline docs/installing-linux.md:36]: System Information should be all lowercase.

System Information -> system information

File: docs/installing-windows.md [hunk 22]
```diff
@@ -41,9 +41,12 @@ If you want to create a Windows installer from ChromeOS, see the [Installing Ven
     6. Right click LabConfig and create new DWORD (32 bit) called "BypassTPMCheck" and set it to 1
     7. Close regedit and command prompt and proceed with installation normally
 
-4. Install drivers:
+3. Install drivers:
    See [Coolstar's Windows Install Helper](https://coolstar.org/chromebook/windows-install.html) for drivers.
 
+4. (Optional) Install Chultrabook Tools for fan controls, backlight controls, System Information and more
```
Comment (meghan06) [inline docs/installing-windows.md:47]: System Information should be all lowercase.

System Information -> system information

File: docs/supported-devices.md [hunk 23]
```diff
@@ -8,14 +8,16 @@ NOTE - do not edit the supported-devices.md file, edit the template file in the
...
 
-The chart below provides a full, authoritative list of all ChromeOS devices currently supported by MrChromebox's Firmware Utility Script, the firmware types available for each, and the hardware write-protect method used. In most cases, all of the devices in a given platform/family are supported identically, but sometimes there are outliers (esp with older or newer devices). The goal is to offer both RW_LEGACY and UEFI Full ROM firmware for all devices wherever possible, so you have the option of dual booting ChromeOS, or liberating your device completely :)
+Be aware that EOL (End Of Life) devices **Do Not** have RW_Legacy avaliable
```
Comment (meghan06) [inline docs/supported-devices.md:20]: Would be better if Do Not was all caps

Do Not -> DO NOT.

Or, it can be all lowercase. either way, the words are bolded so it shouldn't really matter.

File: docs/bootableusb.md (hunk 19 above)
Comment (ethanaobrien) [reply docs/bootableusb.md:53]: definitely is a thing

Comment (death7654) [reply docs/bootableusb.md]: deleted balena etcher and added a reccomendation for linux and macos with ventoy

**Review by death7654 — COMMENTED** [review]
all changes should be added

**Review by ninelore — APPROVED** [review]
Looks good so far, but someone else should proofread again

File: docs/altos.md
Comment (WeirdTreeThing) [inline docs/altos.md:1]: Could just say 11th gen

File: docs/getting-started.md [hunk 24]
```diff
@@ -36,10 +36,12 @@ Please refer to [this chart](supported-devices.html) for a full list of supporte
   * Your iGPU
   * Your storage devices (HDD/SSD, eMMC, NVMe, etc.)
   * **Your boardname**
+       * For example, the boardname for Asus Chromebook C302CA is ```Cave```
   * If you don't know where to find this info, take a look at the [System Info Page.](system-info.html)
 *  **A basic knowledge of command lines and how to use a terminal/command prompt.**
-  * We can't help you if you don't know how to cd to a directory or delete a file.
+  * We cannot help you if you do not know how to go to a directory or delete a file from a terminal.
 * A minimum of an 8GB USB drive or SD Card for creating a bootable OS installer.
+* A Chromebook running Ventoy (read [Ventoy](ventoy.html), or another computer running Balena Etcher, or Rufus (read [Making a Bootable USB](bootableusb.html)
```
Comment (WeirdTreeThing) [inline docs/getting-started.md:44]: remove etcher

File: docs/supported-devices.md [hunk 25]
```diff
@@ -8,14 +8,16 @@ NOTE - do not edit the supported-devices.md file, edit the template file in the
...
 
 ## Supported Devices and Platforms
-
-Supported devices are listed in the table below, grouped by platform/family, roughly in chronological order. To reiterate, support is determined entirely and exclusively by the Board Name. Device Names are just for reference, and may be incomplete as many models/sub-models may use the same Board Name. Figure out the Board Name and then locate it in the table. If it's not listed, it's not supported.
+Supported devices are listed in the table below, grouped by platform/family. If your device is not listed, it is not supported. Support is determined entirely and exclusively by the boardname. Use your Chromebook's boardname to see if it has support. 
 
 ### Determining Device Support
+What's in a name? All ChromeOS devices have a board/device name, which determines which firmware, OS build, etc a device uses. The board name is listed at the bottom of both the Recovery Mode and Developer Mode screens, as part of the hardware ID (HWID).
+
+The chart below provides a full list of all ChromeOS devices currently supported by MrChromebox's Firmware Utility Script, the firmware types available for each, and the hardware write-protect method used.
```
Comment (WeirdTreeThing) [inline docs/supported-devices.md:16]: we also have os support notes

**Review by WeirdTreeThing — CHANGES_REQUESTED** [review]

**Review by WeirdTreeThing — APPROVED** [review]

File: docs/supported-devices.md [hunk 26]
```diff
@@ -13,7 +13,7 @@ Supported devices are listed in the table below, grouped by platform/family. If
 ### Determining Device Support
 What's in a name? All ChromeOS devices have a board/device name, which determines which firmware, OS build, etc a device uses. The board name is listed at the bottom of both the Recovery Mode and Developer Mode screens, as part of the hardware ID (HWID).
 
-The chart below provides a full list of all ChromeOS devices currently supported by MrChromebox's Firmware Utility Script, the firmware types available for each, and the hardware write-protect method used. 
+The chart below provides a full list of all ChromeOS devices currently supported by MrChromebox's Firmware Utility Script, the firmware types available for each, the hardware write-protect method used, and notes for Windows, Linux, and macOS operating systems.
```
Comment (ethanaobrien) [inline docs/supported-devices.md:16]: \

**Review by ethanaobrien — CHANGES_REQUESTED** [review]

File: docs/installing-windows.md [hunk 27]
```diff
@@ -32,7 +32,7 @@ If you want to create a Windows installer from ChromeOS, see the [Installing Ven
    {: .warning }
     **Do NOT** unplug the install USB until the installation has completed. If you do so, the installation will fail as the Windows install file won't exist anymore.
 
-3. Bypassing Windows 11 installation checks:
+2. Bypassing Windows 11 installation checks:
```
Comment (meghan06) [inline docs/installing-windows.md:35]: Also leave a note that people using Rufus can disable the checks from there.

**Review by meghan06 — APPROVED** [review]

**Review by ninelore — APPROVED** [review]

**Comment by ninelore** [comment]
@death7654 please resolve merge conflicts

**Comment by death7654** [comment]
It is ready

**Review by WeirdTreeThing — APPROVED** [review]

**Review by ninelore — APPROVED** [review]
